jenkins-design-language/less/components/material-icon.less

```diff
@@ -1,6 +1,6 @@
 .svg-icon {
-    display: 'inline-block';
```

Comment:
Good catch 👏

Reply:
@vivek does inline-block with quotes behave differently than without quotes?

Reply:
@dragoonis if the display value is written in quotes it is ignored by the browser, like this:
![image](https://user-images.githubusercontent.com/2933351/41963702-93ca4148-7a3b-11e8-8271-5df8377a1147.png)

Reply:
Probably just caused by an oversight when somebody was moving styles from hard-coded react props into CSS.

Reply:
Yeah I did that, just copy-pasted and did not realise at the time that the icons were not vertically aligned (that was the only consequence in this case).